mlflow/store/dbfs_artifact_repo.py

```diff
@@ -98,9 +98,14 @@ def log_artifacts(self, local_dir, artifact_path=None):
                 dir_http_endpoint = self.get_path_module().join(root_http_endpoint, rel_path)
             for name in filenames:
                 endpoint = self.get_path_module().join(dir_http_endpoint, name)
```

Comment:
A lot of this code is common with `log_artifact` -- specifically, each file in this directory are `POST`ed individually to the artifact endpoint. Is is possible to extract the common part of `log_artifact` and `log_artifacts` into a core method to avoid one being updated and not the other?

Also this function is missing `artifact_path == ''` check, like [here in the other func](https://github.com/mlflow/mlflow/blob/e61c89d0fcc5bc7db53baccd5a1502e51ba3f44c/mlflow/store/dbfs_artifact_repo.py#L70)

Reply:
@mparkhe it's a good point - I've updated `log_artifacts` to just call `log_artifact`. I'm not actually not sure why we validated `artifact_path != ''"` in `log_artifact`. Having that validation causes the DBFS artifact repo to behave differently from other artifact repos (i.e. with the validation, it's possible to write code that logs successfully to S3 but breaks when logging to DBFS), so I've removed it for now.